apps/src/aichat/views/chatWorkspace.module.scss

```diff
@@ -2,10 +2,11 @@ $message-editor-height: 100px;
 $default-spacing: 2px;
 
 .chatWorkspace {
+    width: 100%;
+    height: 95%;
```

Comment:
just curious, why this change?

Reply:
I changed `width` to `100%` because I noticed that if the message is relatively short, the width of the workspace narrows.
I changed `height` to `95%` so the text input box can be seen ene if there is an 'Extra Links' box at the the bottom lefthand corner.